ludwig/explain/captum.py

```diff
@@ -71,11 +83,14 @@ def retry_on_cuda_oom_wrapper(*args, **kwargs):
                         # Not a CUDA error
                         raise
 
-            raise RuntimeError("CUDA OOM raised during explanation, but batch size cannot be reduced any further")
+            raise RuntimeError(
+                f"Ran into latest error {latest_error} during explanation. "
```

Comment:
Could we log the full traceback instead of the error message only (I think that's what this string formatting does). That could save us some time when debugging explain errors  by surfacing the relevant code path.